core/amber/src/main/scala/edu/uci/ics/texera/Utils.scala

```diff
@@ -91,4 +92,20 @@ object Utils {
     "www",
     "html"
   )
+
+  def aggregatedStateToString(state: WorkflowAggregatedState): String = {
+    state match {
+      case WorkflowAggregatedState.UNINITIALIZED                   => "Uninitialized"
+      case WorkflowAggregatedState.READY                           => "Initializing"
+      case WorkflowAggregatedState.RUNNING                         => "Running"
+      case WorkflowAggregatedState.PAUSING                         => "Pausing"
+      case WorkflowAggregatedState.PAUSED                          => "Paused"
+      case WorkflowAggregatedState.RESUMING                        => "Resuming"
+      case WorkflowAggregatedState.RECOVERING                      => "Recovering"
+      case WorkflowAggregatedState.COMPLETED                       => "Completed"
+      case WorkflowAggregatedState.ABORTED                         => "Aborted"
+      case WorkflowAggregatedState.UNKNOWN                         => "Unknown"
+      case WorkflowAggregatedState.Unrecognized(unrecognizedValue) => "???"
```

Comment:
make it fail?

Reply:
made it `Unrecognized($unrecognizedValue)` so that it can contain more information